src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/QuestionnaireAnswer.kt

```diff
@@ -51,6 +52,9 @@ data class QuestionnaireAnswer(
   @Convert(converter = YesNoConverter::class)
   val dateRequired: Boolean = false,
 
+  @Column(name = "EXPIRY_DATE")
```

Comment:
The changes to questionnaire tables are just here for completeness, we don't return this data (initially we thought we might) but added to db mappings to ensure that they now match the updated ones in prison-api